lib/src/Uncertainty/Algorithm/Classification/MinimumVolumeClassifier.cxx

```diff
@@ -0,0 +1,248 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief Associate to a given point its class index
+ *
+ *  Copyright 2005-2020 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "openturns/MinimumVolumeClassifier.hxx"
+#include "openturns/PersistentObjectFactory.hxx"
+#include "openturns/Cloud.hxx"
+#include "openturns/Contour.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+CLASSNAMEINIT(MinimumVolumeClassifier)
+
+static const Factory<MinimumVolumeClassifier> Factory_MinimumVolumeClassifier;
+
+/* Default constructor */
+MinimumVolumeClassifier::MinimumVolumeClassifier()
+  : ClassifierImplementation()
+{
+  // Nothing to do
+}
+
+/* Default constructor */
+MinimumVolumeClassifier::MinimumVolumeClassifier(const Distribution & distribution,
+                                                         const Scalar alpha)
+  : ClassifierImplementation()
+  , distribution_(distribution)
```

Comment:
You rely on `computeMinimumVolumeLevelSetWithThreshold` thus we might check continuity here, isn't it?